bin/docker-image-tool.sh

```diff
@@ -63,9 +63,11 @@ function build {
     error "Cannot find docker image. This script must be run from a runnable distribution of Apache Spark."
   fi
 
+  local DOCKERFILE=${DOCKERFILE:-"$IMG_PATH/spark/Dockerfile"}
```

Comment:
Any particular reason why this is tied to this change?

Reply:
Uhm, this technically could be a separate PR. I was initially piggy-backing off the work of Marcelo in being able to build Dockerfiles via -f.

Reply:
Can we separate this out?

Reply:
Not a problem.